src/index.js

```diff
@@ -59,6 +61,35 @@ export default class ImgixClient {
     );
   }
 
+  static _buildOneStepURL({ url, params, opts }) {
```

Comment:
I'm curious what the specific motivation is for accepting these parameters as an object? It seems a little unusual since none of the other instance methods expect their parameters this way.

Reply:
Ah that's a habit of mine more than anything. 

I've gotten object parameters drilled into me from past experience. Purported benefits is that you can never get the order of the arguments wrong and you create fewer breaking changes down the road when you add and or remove parameters. But more than happy to remove this for consistency's sake and stick to normal parameters if that's perferable.